### PR TITLE
Madison/jenkins

### PR DESCRIPTION
**Pull request by madisonmay**

### CODE REVIEW

File: tests/test_general_api.py [hunk 1]
```diff
@@ -64,6 +64,8 @@ def test_multifield_classify(self):
         Ensure saving + loading does not cause errors
         Ensure saving + loading does not change predictions
         """
+        self.model.hparams.batch_size = 2
```
Comment (benleetownsend) [inline tests/test_general_api.py]: We can create a test hparams and pass it at construction time. 

```
def get_test_hparams():
    hparams = get_default_hparams()
    hparams.hvu9rhu90e = hvufe
    ...
    return hparams
```
I think this is a little tidier? I think changing hparams after model construction is reasonably undefined behaviour.

File: finetune/network_modules.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def classifier(hidden, targets, n_classes, dropout_placeholder, hparams, train=F
     with tf.variable_scope('model', reuse=reuse):
         hidden = dropout(hidden, hparams.clf_p_drop, train, dropout_placeholder)
         clf_logits = mlp(hidden, n_classes, hparams)
-        clf_losses = tf.nn.softmax_cross_entropy_with_logits(logits=clf_logits, labels=targets)
+        clf_losses = tf.nn.softmax_cross_entropy_with_logits_v2(logits=clf_logits, labels=targets)
```
Comment (benleetownsend) [inline finetune/network_modules.py]: Worth putting `tf.stop_gradient` to maintain consistency? At the moment we dont have any reason to do so. But if no gradients flow to the targets its 0 cost. In the future maybe we put something in that it does effect and not realize?

Comment (madisonmay) [reply finetune/network_modules.py]: Yea probably.  Added that in.
